likecoin/public/likecoin-button.php

```diff
@@ -53,14 +53,18 @@ function likecoin_add_likebutton( $likecoin_id = '' ) {
 	$post_type_query = ''; // empty query means any type.
 	$widget_position = 'bottom'; // default to bottom.
 
+	$per_post_widget_position = '';
 	// follow post meta only if per post option is set.
 	if ( isset( $option[ LC_OPTION_BUTTON_DISPLAY_AUTHOR_OVERRIDE ] ) && $option[ LC_OPTION_BUTTON_DISPLAY_AUTHOR_OVERRIDE ] ) {
 		$widget_option = get_post_meta( $post->ID, LC_OPTION_WIDGET_OPTION, true );
 		// default to 'bottom' if nothing is set, since liker id is set.
 		if ( isset( $widget_option[ LC_OPTION_WIDGET_POSITION ] ) ) {
-			$widget_position = $widget_option[ LC_OPTION_WIDGET_POSITION ];
+			$per_post_widget_position = $widget_option[ LC_OPTION_WIDGET_POSITION ];
 		}
-	} else { // follow site setting.
+	}
+	if ( isset( $per_post_widget_position ) && ! empty( $per_post_widget_position ) ) {
```

Comment:
Curious about if following change would cause logical difference? 
```suggestion
	if ( ! empty( $per_post_widget_position ) ) {
```

Reply:
isset checks if it was actually set, where empty might throw warning if var not set.
I think logically we won't have not set case due to we already checked isset above for widget_option[ LC_OPTION_WIDGET_POSITION ]; , but this is php so I don't dare to not check again here, esp if we might change above later

Reply:
I see, thanks!